curriculum/challenges/english/blocks/workshop-caesar-cipher/680a549f60f5a3fedfc0edd9.md

```diff
@@ -23,7 +23,7 @@ As a reminder, `sentence[start:stop]` returns the characters of `sentence` from
 
 # --hints--
 
-You should concatenate the missing first portion of `alphabet` to `alphabet[shift:]` using the slicing syntax. Use `shift` to specify where to stop the slicing.
+You should assign the concatenation of `alphabet[shift:]` and the missing first portion of `alphabet` to the `shifted_alphabet` variable. Use `shift` to specify where to stop the slicing.
```

Comment:
the hint make it seem that it should be in the opposite order as the expected solution, can you reword it?